source-container-build/app/requirements-build.txt

```diff
@@ -0,0 +1,10 @@
+#
+# This file is autogenerated by pip-compile with Python 3.12
+# by the following command:
+#
+#    pybuild-deps compile --generate-hashes --output-file=requirements-build.txt app/requirements.txt
+#
+poetry-core==2.1.3 \
+    --hash=sha256:0522a015477ed622c89aad56a477a57813cace0c8e7ff2a2906b7ef4a2e296a4 \
+    --hash=sha256:2c704f05016698a54ca1d327f46ce2426d72eaca6ff614132c8477c292266771
+    # via backoff
```

Comment:
What is this file used for?

Reply:
python build time dependencies for packages